cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/cli/ExtractionCommand.java

```diff
@@ -39,6 +59,21 @@ public void run() {
       try {
         final JacksonJsonProvider reader = new JacksonJsonProvider();
         final IngestConfig context = reader.toObject(file, IngestConfig.class);
+        if (context != null) {
+          InputConfig inputConfig = context.getInput();
+          IIIFConfig iiifConfig = inputConfig.getIiif();
+          if (iiifConfig != null) {
+            String directoryPath;
+            String iiifConfigPath = inputConfig.getPath();
+            if (iiifConfigPath == null || iiifConfigPath.isEmpty()) {
+              directoryPath = file.getAbsoluteFile().toPath().getParent() + "/iiif-media";
```

Comment:
The target should probably be configurable, rather than being hard-coded to 'iiif-media'.

Reply:
Contrary to what I said in the meeting yesterday, this is a feature not a bug. The `/iiif-media` path is a fallback which used only when the user has not specified a directory in the config file.